internal/services/paloalto/schema/network_profile.go

```diff
@@ -67,6 +70,18 @@ func VnetNetworkProfileSchema() *pluginsdk.Schema {
 					},
 				},
 
+				"trusted_ranges": {
```

Comment:
should this be
```suggestion
				"trusted_addresses_ranges": {
```
or
```suggestion
				"trusted_addresses": {
```
to both be more clear what it is looking for as well match other propertyes int he provider?

Reply:
Updated